### PR TITLE
Security integration, docker fixes, SDK-302

### PR DESCRIPTION
**Pull request by vol4onok**

### CODE REVIEW

File: infrastructure/installer.sh [hunk 1]
```diff
@@ -23,11 +23,25 @@ tail -n+"${ARCHIVE}" "${0}" | tar xpJ -C "${DESTINATION}"
 ${DESTINATION}/bin/spryker-sdk.sh sdk:init:sdk
 ${DESTINATION}/bin/spryker-sdk.sh sdk:update:all
 
-echo ""
-echo "Installation complete."
-echo "To use the spryker sdk execute: "
-echo "echo \"alias spryker-sdk='${DESTINATION}/bin/spryker-sdk.sh'\" >> ~/.bashrc && source ~/.bashrc OR echo \"alias spryker-sdk='</path/to/install/sdk/in>/bin/spryker-sdk.sh'\" >> ~/.zshrc  && source ~/.zshrc if you use zsh"
-echo ""
+if [ ! -f "$FILE" ]
```
Comment (gechetspr) [inline infrastructure/installer.sh:26]: I assume we don't need this section

File: infrastructure/installer.sh [hunk 2]
```diff
@@ -23,11 +23,25 @@ tail -n+"${ARCHIVE}" "${0}" | tar xpJ -C "${DESTINATION}"
...
+then
+    echo "File $FILE does not exist"
+fi
+
+if [ -f ~/.bashrc ]
+then
+    echo "alias spryker-sdk2=\"${DESTINATION}/bin/spryker-sdk.sh\"" >> ~/.bashrc && source ~/.bashrc
```
Comment (gechetspr) [inline infrastructure/installer.sh:33]: spryker-sdk instead of spryker-sdk2

File: src/Core/Appplication/Service/TaskExecutor.php [hunk 3]
```diff
@@ -178,13 +178,12 @@ protected function executeStage(ContextInterface $context, string $stage = Conte
 
             foreach ($task->getCommands() as $command) {
                 $context = $this->commandExecutor->execute($command, $context, $task->getId());
-                $commands[] = $task->getCommands();
-                if ($context->getExitCode() !== 0 && $command->hasStopOnError()) {
+                $commands[] = $command;
+                if ($context->getExitCode() !== 0 && $command->hasStopOnError()) {echo '+++';
```
Comment (gechetspr) [inline src/Core/Appplication/Service/TaskExecutor.php:182]: this is leftover?

File: src/Presentation/Console/Resources/config/services.yaml [hunk 4]
```diff
@@ -42,7 +42,9 @@ services:
     tags: ['console.command']
     arguments:
       - '@ide.config_manager'
-
+  security_checker_command:
+    class: SecurityChecker\Command\SecurityCheckerCommand
```
Comment (gechetspr) [inline src/Presentation/Console/Resources/config/services.yaml:46]: but what about task file? don't we need it?

Comment (vol4onok) [reply src/Presentation/Console/Resources/config/services.yaml:46]: we don't we already have command and don't have any other entrypoints for this command

**Review by gechetspr — APPROVED** [review]

**Pull request merged**